public/regdesk.html

```diff
@@ -53,23 +53,14 @@
         </div>
       </div>
     </nav>
-    <div class="row row-no-margin container-no-gutter">
-      <div id="buttonGutterLeft" class="col-2 d-grid gap-2 container-no-gutter">
-        <button id="cancelButton" class="btn btn-danger btn-fullheight">CANCEL</button>
+    <div class="row row-no-margin container-no-gutter h-100">
+      <div id="buttonGutterLeft" class="col-2 d-grid gap-2 h-100 container-no-gutter-left">
+        <button id="leftButton" class="btn btn-danger btn-fullheight btn-huge">CANCEL</button>
```

Comment:
I think this was more accurate as cancelButton.

Reply:
Good point, rolled it back but with "btn" to match the CSS selector convention.